capstone/src/main/java/capstone/PopulateData.java

```diff
@@ -0,0 +1,143 @@
+                Date.valueOf("2023-09-14"),
+                "Shipyard",
+                "99999904")
+            .get();
+    /*** Creates customer_5 and store into the database */
+    Customer customer_5 =
+        Database.createCustomer(
+                "customer_5",
+                "password",
+                "Customer",
+                "5",
+                "S99000005",
+                "customer_5@cust.com",
+                Date.valueOf("2023-09-15"),
+                "Yacht",
+                "99999905")
+            .get();
+
+    /*** Creates an account for customer_1 and store into the database */
+    Account cust_acc_1 = Database.createAccount(customer_1, AccountType.CURRENT).get();
+    /*** Creates an account for customer_2 and store into the database */
+    Account cust_acc_2 = Database.createAccount(customer_2, AccountType.SAVINGS).get();
+    /*** Creates an account for customer_3 and store into the database */
+    Account cust_acc_3 = Database.createAccount(customer_3, AccountType.CURRENT).get();
+    /*** Creates an account for customer_4 and store into the database */
+    Account cust_acc_4 = Database.createAccount(customer_4, AccountType.SAVINGS).get();
+    /*** Creates an account for customer_5 and store into the database */
+    Account cust_acc_5 = Database.createAccount(customer_5, AccountType.CURRENT).get();
+
+    /*** Creates a transaction and credits into customer_1 account */
+    Transaction transaction_1 =
+        Database.createTransaction(customer_1, cust_acc_1, TransactionType.CREDIT, 500).get();
+    Database.updateBalance(cust_acc_1, transaction_1.getAmount());
+
+    /*** Creates a 'debit' transaction from the customer_1 account */
+    Transaction transaction_2 =
+        Database.createTransaction(customer_1, cust_acc_1, TransactionType.DEBIT, 300).get();
+    Database.updateBalance(cust_acc_1, transaction_2.getAmount());
+
+    /*** Creates a transaction and credits into customer_2 account */
+    Transaction transaction_3 =
+        Database.createTransaction(customer_2, cust_acc_2, TransactionType.CREDIT, 1000).get();
+    Database.updateBalance(cust_acc_2, transaction_3.getAmount());
+
+    /*** Creates a transaction and credits into customer_3 account */
+    Transaction transaction_4 =
+        Database.createTransaction(customer_3, cust_acc_3, TransactionType.CREDIT, 2020).get();
+    Database.updateBalance(cust_acc_3, transaction_4.getAmount());
+
+    /*** Creates a transaction and credits into customer_4 account */
+    Transaction transaction_5 =
+        Database.createTransaction(customer_4, cust_acc_4, TransactionType.CREDIT, 6969).get();
+    Database.updateBalance(cust_acc_4, transaction_5.getAmount());
+
+    /*** Creates a transaction and credits into customer_5 account */
+    Transaction transaction_6 =
+        Database.createTransaction(customer_5, cust_acc_5, TransactionType.CREDIT, 10).get();
+    Database.updateBalance(cust_acc_5, transaction_6.getAmount());
```

Comment:
newBalance = balance + or - getAmount() depending on the transaction type not just the transaction amount 
because updateBalance sets the amount rather than deducting or adding the amount.